backend/server/controllers/user.js

```diff
@@ -1,72 +1,73 @@
-import models from '../../database/models';
-import { Authentication } from '../../middlewares/authentication';
+import models from '../../database/models'
+import { Authentication } from '../../middlewares/authentication'
+import { UserDTO } from '../../dto/users'
 
 export const addUser = async (req, res) => {
-  try {
-    const { body } = req;
-    console.log('🚀 ~ file: user.js ~ line 6 ~ addUser ~ body', body);
+    try {
+        const { body } = req
+        console.log('🚀 ~ file: user.js ~ line 6 ~ addUser ~ body', body)
 
-    const createUser = await models.users.create({
-      username: body.username,
-      email: body.email,
-      password: body.password
-    });
+        const createUser = await models.users.create({
+            username: body.username,
+            email: body.email,
+            password: body.password,
+        })
 
-    res.status(201).send(createUser);
-  } catch (error) {
-    res.status(500).send(error);
-  }
-};
+        res.status(201).send(createUser)
```

Comment:
this needs to be converted to dto as well before being returned, also can you add a test for this endpoint as well?